test/Microsoft.AspNetCore.Razor.Test.MvcShim.Version1_X/Microsoft.AspNetCore.Razor.Test.MvcShim.Version1_X.csproj

```diff
@@ -1,7 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+    <!-- 
+      DO NOT change this to $(StandardTestTfms) - this is a dependency that mimics MVC - it does not contain tests.
+    -->
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
This starts breaking lots of things if it targets netcoreapp21

Reply:
Since this is supposed to **look like** MVC it makes much more sense if it's supposed to be netstandard20